lib/cc/analyzer/formatters/html_formatter.rb

```diff
@@ -1,34 +1,280 @@
 require "redcarpet"
-require "active_support/number_helper"
 
 module CC
   module Analyzer
     module Formatters
-      class HTMLFormatter < Formatter
+      class HTMLFormatter < Formatter # rubocop: disable Metrics/ClassLength
+        LANGUAGES = Hash.new { |_, ext| ext }.
+                    merge(
+                      # abap
```

Comment:
Ahh, right, I saw that default-val code but didn't make the connection. Thanks for the explanation.